FurCoNZ/Controllers/OrderController.cs

```diff
@@ -50,11 +50,40 @@ public async Task<IActionResult> Index(OrderIndexViewModel orderIndexViewModel,
             if (ModelState.IsValid)
```

Comment:
What happens when no tickets are ordered?

Reply:
Not handled yet. I'll fix this up when I do the validation work in a later feature branch. I want to get something going roughly for now.